### PR TITLE
[Metadata] Fix PDL files using AuditStamp type with default values

### PR DESCRIPTION
**Pull request by pedro93**
Removes AuditStamp default values in PDL files to make the generated Avro schemas parseable.

Fixes the following stack trace when trying to consume MAE events using Confluent Kafka Connect:
```
Caused by: org.apache.kafka.connect.errors.DataException: Invalid class for string type, expecting String or CharSequence but found class org.apache.avro.JsonProperties$Null
	at io.confluent.connect.avro.AvroData.toConnectData(AvroData.java:1390)
```

### Details
The default values in the PDL files were not specifying values for all fields of the AuditStamp type, this caused pegasus to generate avro schemas that filled in the missing properties. In this case `impersonator` field was missing, which lead to the following avro structure:
```
{
   "type":"record",
   "name":"EditableChartProperties",
   "namespace":"com.linkedin.chart",
   "doc":"Stores editable changes made to properties. This separates changes made from\ningestion pipelines and edits in the UI to avoid accidental overwrites of user-provided data by ingestion pipelines",
   "fields":[
      {
         "name":"created",
         "type":"com.linkedin.common.AuditStamp",
         "doc":"An AuditStamp corresponding to the creation of this resource/association/sub-resource. A value of 0 for time indicates missing data.",
         "default":{
            "actor":"urn:li:corpuser:unknown", # <---- Explicitly set in PDL file
            "impersonator":null,  # <---- NOT set in PDL file
            "time":0  # <---- Explicitly set in PDL file
         }
      },...
   ]
}
```

_Note_ the default value for `com.linkedin.common.AuditStamp` type's "impersonator" field.
This field's schema is defined as a nullable string:
```
{
   "name":"impersonator",
   "type":[
      "null",
      "string"
   ],
   "doc":"The entity (e.g. a service URN) which performs the change on behalf of the Actor and must be authorized to act as the Actor.",
   "default":null,
   "java":{
      "class":"com.linkedin.common.urn.Urn"
   }
}
```

However, the avro parser interprets `null` as being a json property. 
For more information see this [Avro Issue](https://issues.apache.org/jira/browse/AVRO-1954). 
Reportedly fixed on avro v1.9.0 but we are using KafkaAvroSerializer v5.5.1 which uses Avro v1.9.2 so something seems to be amiss. The workaround in the issue states to remove the default value from the "defaults" property.

### Test Plan
Regression unit test added.

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -76,6 +76,8 @@ project.ext.externalDependency = [
     // avro-serde includes dependencies for `kafka-avro-serializer` `kafka-schema-registry-client` and `avro`
     'kafkaAvroSerde': 'io.confluent:kafka-streams-avro-serde:5.5.1',
     'kafkaClients': 'org.apache.kafka:kafka-clients:2.3.0',
+    'kafkaAvroConnect': 'io.confluent:kafka-connect-avro-data:5.5.1',
```
Comment (dexter-mh-lee) [inline build.gradle:79]: Are these needed?

**Review by dexter-mh-lee — APPROVED** [review]
LGTM! One minor comment

File: build.gradle (hunk 1 above)
Comment (pedro93) [reply build.gradle:79]: Yes, it is in these confluent modules that our serialization breaks when using confluent software (Kafka connect) to read our avro events.

I use them in the unit test, you can check there.

**Comment by shirshanka** [comment]
This is a breaking change. Closing this until we find a different way.

**Pull request closed without merging**